conf/parsers.conf

```diff
@@ -83,10 +83,10 @@
     Time_Key time
 
 [PARSER]
-    # http://rubular.com/r/izM6olvshn
-    Name crio
+    # http://rubular.com/r/tjUt3Awgg4
+    Name cri
     Format Regex
```

Comment:
We could make this `regex` although there was a commit to make this case insensitive merged.